tracer/src/Datadog.Trace/AppSec/Waf/Waf.cs

```diff
@@ -122,7 +122,7 @@ public bool ToggleRules(IDictionary<string, bool> ruleStatus)
                 return false;
             }
 
-            var encoded = encoder.Encode(ruleStatus);
+            using var encoded = encoder.Encode(ruleStatus);
```

Comment:
Maybe you want to also include `using` here:
https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs#L175

Reply:
this one we return it and need it for a later stage to configure the waf [here ](https://github.com/DataDog/dd-trace-dotnet/blob/a8abc21d58ea9d91bc0b5b2ba5f9d265f97e41d9/tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs#L110)